src/Infra/MySQL/MySQLItemRepository.ts

```diff
@@ -7,6 +7,19 @@ import { ItemRepositoryInterface } from '../../Domain/Item/ItemRepositoryInterfa
 @injectable()
 @EntityRepository(Item)
 export class MySQLItemRepository extends Repository<Item> implements ItemRepositoryInterface {
+  async findDatesForComputingIntegrityHash(userUuid: string): Promise<number[]> {
+    const queryBuilder = this.createQueryBuilder('item')
+    queryBuilder.select('item.updated_at_timestamp')
+    queryBuilder.where('item.user_uuid = :userUuid', { userUuid: userUuid })
+    queryBuilder.where('item.deleted = :deleted', { deleted: false })
+    queryBuilder.where('item.content_type IS NOT NULL')
+    queryBuilder.orderBy('item.updated_at_timestamp', 'DESC')
```

Comment:
To be sure: so the query builder produced by `createQueryBuilder` is stateful and the lines above will mutate the same object rather than producing updated versions?

Reply:
I think this will indeed happen.

`where` docs say:

```
calling this function will override previously set WHERE conditions.
```

so I think it should be `andWhere`:

```
Adds new AND WHERE condition in the query builder.
```

Reply:
Good catch - will fix 👍